stl/inc/chrono

```diff
@@ -121,9 +121,8 @@ namespace chrono {
             : _MyRep(static_cast<_Rep>(_Val)) {}
 
         template <class _Rep2, class _Period2,
-            enable_if_t<
-                treat_as_floating_point_v<
-                    _Rep> || (_Ratio_divide_sfinae<_Period2, _Period>::den == 1 && !treat_as_floating_point_v<_Rep2>),
+            enable_if_t<treat_as_floating_point_v<
+                            _Rep> || (ratio_divide<_Period2, _Period>::den == 1 && !treat_as_floating_point_v<_Rep2>),

```

Comment:
You should be able to salvage that clang format issue by adding a `\\` after `_Rep>`

Reply:
I don't think there's a digraph for `\\` and I guess `//` was meant

Reply:
\\ Ups